server/views/licenceConditions/includes/courseOrCentre.pug

```diff
@@ -5,7 +5,7 @@ div.pure-g.paddingBottom
       if condition.user_submission && condition.user_submission.courseOrCentre
         | #{condition.user_submission.courseOrCentre}
 
-
+- var behaviours = condition.user_submission && condition.user_submission.abuseAndBehaviours || []
```

Comment:
rather than reading from the template data we can now read these directly from the user submission data as single values are properly handled by the `displayForEdit` function